helm-chart/values.yaml

```diff
@@ -0,0 +1,150 @@
+# Global settings
+global:
+  storageClass: ""
+  namespace: gravitino-playground
+
+# Hive settings
+hive:
+  image:
+    repository: datastrato/hive
+    tag: 2.7.3-no-yarn
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 3
+      memory: 3Gi
+    requests:
+      cpu: 1
+      memory: 1Gi
+
+# Gravitino settings
+gravitino:
+  image:
+    repository: datastrato/gravitino
+    tag: 0.5.1
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+
+# Trino settings
+trino:
+  image:
+    repository: datastrato/trino
+    tag: 435-gravitino-0.5.1
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 2Gi
+    requests:
+      cpu: 500m
+      memory: 1Gi
+
+# PostgreSQL settings
+postgresql:
+  image:
+    repository: postgres
+    tag: "13"
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+  persistence:
+    enabled: true
+    size: 1Gi
+
+# MySQL settings
+mysql:
+  image:
+    repository: mysql
+    tag: "8.0"
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+  persistence:
+    enabled: true
+    size: 1Gi
+
+# Spark settings
+spark:
+  image:
+    repository: spark
+    tag: 3.4.1-scala2.12-java11-ubuntu
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 1Gi
+    requests:
+      cpu: 500m
+      memory: 500Mi
+
+# Jupyter settings
+jupyter:
+  image:
+    repository: jupyter/minimal-notebook
+    tag: latest
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+  persistence:
+    enabled: true
+    size: 1Gi
+
+# Service specific settings
+service:
+  type: ClusterIP
```

Comment:
There is something wrong when I change the service type to `LoadBalancer`
```
ubuntu@ip-10-0-4-171:~$ helm upgrade --install gravitino-playground ./gravitino-playground --create-namespace --namespace gravitino-playground
Release "gravitino-playground" does not exist. Installing it now.
NAME: gravitino-playground
LAST DEPLOYED: Tue Aug  6 10:17:46 2024
NAMESPACE: gravitino-playground
STATUS: deployed
REVISION: 1
NOTES:
1. Get the application URL by running these commands:
     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
           You can watch the status of by running 'kubectl get --namespace gravitino-playground svc -w gravitino-playground'
  export SERVICE_IP=$(kubectl get svc --namespace gravitino-playground gravitino-playground --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}")
  echo http://$SERVICE_IP:
ubuntu@ip-10-0-4-171:~$ kubectl get svc --namespace gravitino-playground gravitino-playground --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"
Error from server (NotFound): services "gravitino-playground" not found
```

Reply:
BTW, can you add an example about the ingress enabled scenario in README.md?

Reply:
Do we need to support ingress ?

Reply:
Ingress is not necessary.